lib/locale/pl_PL/strings.php

```diff
@@ -24,6 +24,7 @@
  *  $Id$
  */
 
+$_LANG['Nothing has changed in positions of the document!'] = 'W pozycjach dokumentu nic się nie zmieniło!';
```

Comment:
Nowe "stringi" umieszczamy na górze, gdy mają szansę być przenoszone do wydań LMS+. Gdy tego nie przewidujemy ( a tak pewnie z tym będzie, że nie trafi do LMS+, bo za bardzo inwazyjne) to lepiej umieszczać wg alfabetu (wówczas prawdopodobieństwo konfliktu między poszczególnymi gałęziami zmniejsza się).

Reply:
Ok. ale przecież wpisy w pliku nie są ułożone alfabetycznie, to gdzie to umieścić?

Reply:
Dopóki mocno pilnowałem porządku to były. Poszukaj gdzieś w środku ciągu fraz na literę `N` i tam umieść zgodnie z porządkiem porządkowania tekstowego.

Reply:
Poprawiłem.